manifest.json

```diff
@@ -5,7 +5,7 @@
     "manifest_version": 2,
     "content_scripts": [
         {
-            "matches": [ "https://github.com/*" ],
+            "matches": [ "https://*.github/" ],
```

Comment:
So are you sure this works as you describe?

During my testing it did not, and according to Firefox and Chrome this only works if you want to match any _subdomain_ of a given host, not any subdomain of any host.

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Match_patterns#host

Reply:
Yes that's right, it only matches `github.*` which works for my corp GHE, but possibly not yours if it has a different hostname!

If you're happy to open this up to any host on any subdomain, I can use `<all_urls>` here instead?

Thanks for the pointer to the Refined GitHub approach, I found more detail [here](https://github.com/fregante/webext-dynamic-content-scripts/blob/251bf0ccf83f1ca4b1f5969f5d26c3d4da1bc4d7/how-to-add-github-enterprise-support-to-web-extensions.md). I can try that too.

Reply:
> Yes that's right, it only matches `github.*` which works for my corp GHE, but possibly not yours if it has a different hostname!

Yep, also we'll need this to continue to work on github.com as well 😊 

> If you're happy to open this up to any host on any subdomain, I can use `<all_urls>` here instead?

I don't think that would be a good idea, since people that use this extension probably wouldn't appreciate it if I asked for access to all websites they visit 😅 

> Thanks for the pointer to the Refined GitHub approach, I found more detail here. I can try that too.

I think this would be the best solution 👍

Reply:
I've pushed a version that uses [webext-dynamic-content-scripts](https://github.com/fregante/webext-dynamic-content-scripts/blob/master/how-to-add-github-enterprise-support-to-web-extensions.md)

It required adding 2 scripts and an icon to make the drop-down menu appear in Firefox.